packages/webview_flutter/example/lib/main.dart

```diff
@@ -47,3 +62,40 @@ class SampleMenu extends StatelessWidget {
     );
   }
 }
+
+class MoveControl extends StatelessWidget {
+  const MoveControl(this._webViewController)
+      : assert(_webViewController != null);
+
+  final WebViewController _webViewController;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: <Widget>[
+        IconButton(
+            icon: const Icon(Icons.arrow_back_ios),
+            onPressed: () async {
+              final bool canGoBack = await _webViewController.canGoBack;
+              if (canGoBack) {
+                await _webViewController.goBack();
```

Comment:
I think we don't need to await this call

Reply:
Exactly...! -> a3219e2